enterprise/suppress-user/factory_test.go

```diff
@@ -0,0 +1,141 @@
+package suppression
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
+	"github.com/rudderlabs/rudder-go-kit/config"
+	"github.com/rudderlabs/rudder-go-kit/logger"
+	"github.com/rudderlabs/rudder-server/admin"
+	backendconfig "github.com/rudderlabs/rudder-server/backend-config"
+	"github.com/rudderlabs/rudder-server/enterprise/suppress-user/model"
+	"github.com/rudderlabs/rudder-server/services/diagnostics"
+	"github.com/rudderlabs/rudder-server/utils/misc"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	namespaceID = "spaghetti"
```

Comment:
Why not bucatini? :laughing: :stuck_out_tongue_winking_eye:

Reply:
good point, removing it all together. Since, the complete multi-tenant endpoint is useless.